scripts/main/sidebar.js

```diff
@@ -117,6 +117,15 @@ sidebar.changeAttr = function(attr, value = '-', dangerouslySetInnerHTML = false
 
 };
 
+sidebar.secondsToHMS = function(d) {
+	d = Number(d);
+	var h = Math.floor(d / 3600);
+	var m = Math.floor(d % 3600 / 60);
+	var s = Math.floor(d % 3600 % 60);
```

Comment:
```php
// we have
d = h * 3600 + m * 60 + s
d % 3600 = m * 60 + s
d % 3600 % 60 = s
// but also
d = (h * 60 + m ) * 60 + s
d % 60 = s
// thus
d % 3600 % 60 = d % 60
```
:)

Reply:
That's what happens when I copy code from Stackoverflow without thinking... :wink: Good catch!

Reply:
Done